locales/en/fields.json

```diff
@@ -1,8 +1,12 @@
 {
   "date": {
-    "hint": "For example 28/10/1990 or 28 October 1990",
+    "hint": "For example 28/11/2020 or 28 November 2020",
```

Comment:
So going on what we just discussed, I think we should update this one to be `For example 28/10/2020, 28 October 2020 or next Friday`

And you could adapt the DOB field to use `For example 28/10/1990, 28 October 1990 or next Friday` as the 1990 might be more relevant for historic dates like @Andy-Millar suggested.

I think it _should_ just pickup the override hint field for the dob field. If it's not that simple leave it for now.

Reply:
I don't think there's any need for the 'next friday' in the historic one ;)